packages/components/tests/integration/components/hds/tabs/index-test.js

```diff
@@ -7,12 +7,93 @@ module('Integration | Component | hds/tabs/index', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders the component', async function (assert) {
-    await render(hbs`<Hds::Tabs />`);
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
     assert.dom(this.element).exists();
   });
 
   test('it should render with a CSS class that matches the component name', async function (assert) {
-    await render(hbs`<Hds::Tabs id="test-tabs" />`);
+    await render(hbs`
+      <Hds::Tabs id="test-tabs" as |T|>
+        <T.Tab>One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
     assert.dom('#test-tabs').hasClass('hds-tabs');
```

Comment:
Ok but would I test for those doing something like this?
`assert.dom('#test-tabs .hds-tabs__tab:first-child').hasClass('hds-tabs__tab');`

Because that seems kind of redundant. Although I can add those if you think it's helpful. (I could add an id to a `<Tab />` component to find it that way instead but if I add an id to a `<Panel />` component then it would interfere with the id it already uses for some of it's functinality.)

Reply:
I'll use data-test attributes for these like you suggested below.

Reply:
plus, now that you're adding `icon` and `count` to the `Tab`, maybe you could consider splitting the test file in three: one for the Tabs, one for the Tab, and one for the Panel (similar to what it's done for the form controls)